### PR TITLE
PLPO-7

### PR DESCRIPTION
**Pull request by lkulig**

### CODE REVIEW

File: planning-poker-core/src/main/java/com.futureprocessing.planningpoker/Application.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package com.futureprocessing.planningpoker;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+import java.util.Arrays;
+
+@SpringBootApplication
+public class Application {
```
Comment (pbenecki) [inline planning-poker-core/src/main/java/com.futureprocessing.planningpoker/Application.java]: Isn't it duplicated also in planning-poker-app? Also path of this file is a bit strange: src/main/java/com.futureprocessing.planningpoker/Application.java

Comment (pbenecki) [inline planning-poker-core/src/main/java/com.futureprocessing.planningpoker/Application.java]: Clarified f2f.

Comment (lkulig) [inline planning-poker-core/src/main/java/com.futureprocessing.planningpoker/Application.java]: fixed in new commit

File: planning-poker-core/src/main/java/com/futureprocessing/planningpoker/Application.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import org.springframework.context.ApplicationContext;
+import java.util.Arrays;
+
+@SpringBootApplication
+public class Application {
+
+    public static void main(String[] args) {
+        ApplicationContext ctx = SpringApplication.run(Application.class, args);
+
+        System.out.println("Let's inspect the beans provided by Spring Boot:");
```
Comment (akawka) [inline planning-poker-core/src/main/java/com/futureprocessing/planningpoker/Application.java]: Wat?

Comment (dczyz) [inline planning-poker-core/src/main/java/com/futureprocessing/planningpoker/Application.java]: Senior developer...

**Pull request merged**